flake-lang/flake-rust.nix

```diff
@@ -1,33 +1,68 @@
 pkgs:
 
-{ crane, src, crateName, extraSources ? [ ], extraSourcesDir ? ".extras", data ? [ ], dataDir ? "data", devShellHook ? "", devShellTools ? [ ], testTools ? [ ] }:
+{ crane
+, src
+, crateName
+, rustVersion ? "latest"
+, nativeBuildInputs ? [ ]
+, extraSources ? [ ]
+, extraSourcesDir ? ".extras"
```

Comment:
Should we allow the user to change this? It seems the `sed` regex below is hardcoded to check in the `.extras` directory.

Reply:
hmm, that's a nice catch...  I'll use this value in the regex directly